### PR TITLE
Changed jsonSerializer and migrate valakas sounds

### PR DESCRIPTION
**Pull request by Alexanderus32**

### CODE REVIEW

File: MusicKeyStrokes/JsonSerializer.cs [hunk 1]
```diff
@@ -5,20 +5,33 @@
...
 {
     class JsonSerializer
     {
+        private const string defaultMusicKeyFile = "music/musicKey.json";
 
-        public static void addJsonDocumentKey(Key key)
+        public List<T> Deserialize<T>(string fileName = defaultMusicKeyFile)
         {
-            using (FileStream fs = new FileStream("keyList.json", FileMode.OpenOrCreate))
+            if (!File.Exists(fileName))
```
Comment (HagBcemu) [inline MusicKeyStrokes/JsonSerializer.cs:18]: FileMode OpenOrCreate - create file when did't created or open when it created. Using wich FileStream it is cool 
decision

File: MusicKeyStrokes/Form1.cs [hunk 2]
```diff
@@ -16,5 +17,20 @@ public Form1()
...
+            //using serializer 
+            JsonSerializer sr = new JsonSerializer();
+            var music = sr.Deserialize<KeyModel>();
+
+            if (music == null)
+                music = new List<KeyModel>();
+
+            var model = new KeyModel() { NameSound = "ban", KeyValue = Keys.Q, Layout = LayoutSound.Valakas1,PathSound = "music/valakas1/ban.mp3" };
+            music.Add(model);
+            sr.Serialize<KeyModel>(music);
```
Comment (HagBcemu) [inline MusicKeyStrokes/Form1.cs:32]: [https://i.kym-cdn.com/photos/images/newsfeed/001/303/345/54e.jpg](url)

Comment (Alexanderus32) [reply MusicKeyStrokes/Form1.cs:32]: its just for sample. Will be deleted later

File: MusicKeyStrokes/JsonSerializer.cs (hunk 1 above)
Comment (Alexanderus32) [reply MusicKeyStrokes/JsonSerializer.cs:18]: FileStream reads a stream of bytes, and StreamReader reads a string, I think StreamReader is more suitable for us. Иге I don’t know how to create a file if it does not exist in StreamReader.

**Pull request merged**